nl/guides/for-students/exercises/README.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Oefeningen handleiding
+description: "Tutorial: oefeningen handleiding"
+---
+
+# Oefeningen handleiding
+> Alle informatie zodat je als student kan navigeren naar oefeningen of oplossingen, oplossingen kan indienen en feedback kan interpreteren.
+
+## Navigeren naar een oefening
```

Comment:
Rework section. The main point should be that exercises can occur in 2 ways on Dodona:
- embedded in an exercise series of a course
- not in a course, using the /exercises link

This should be written as a user manual and not as technical documentation